ui/src/views/admin/settings/SettingsView.vue

```diff
@@ -1,82 +1,145 @@
-                </v-col>
-
-
-                <v-col
-                  cols="12"
-                >
-                  <v-btn
-                    color="primary"
-                    min-width="150"
+          <v-row justify="center">
+            <v-col cols="4">
+              <v-img
+                class="rounded-circle d-inline-block mx-2 mb-5"
+                :src="require('@assets/svg/profile_picture.svg')"
+                width="128"
+              />
+            </v-col>
+          </v-row>
+          <ValidationObserver v-slot="{ handleSubmit }">
+            <v-form
+              v-model="isFormValid"
+              class="admin-settings-form"
+              @keyup.native.enter="handleSubmit(onFormSubmit)"
+              @submit.prevent="handleSubmit(onFormSubmit)"
+            >
+              <v-container class="py-0">
+                <v-row>
+                  <v-col
+                    cols="12"
+                    class="my-0 py-0"
                   >
-                    Update Profile
-                  </v-btn>
-                </v-col>
-              </v-row>
-            </v-container>
-          </v-form>
-        </material-card>
+                    <ValidationProvider
+                        v-slot="{ errors }"
+                        name="E-mail"
+                        rules="required|email"
+                    >
+                      <v-text-field
+                        v-model.trim="formData.email"
+                        outlined
+                        required
+                        disabled
+                        :error-messages="errors[0]"
+                        label="E-mail"
+                        class="my-0 py-0"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+                  <v-col
+                    cols="12"
+                    class="my-0 py-0"
+                  >
+                    <ValidationProvider
+                        v-slot="{ errors }"
+                        name="Password"
+                        rules="required|min:8"
+                    >
+                      <v-text-field
+                        v-model.trim="formData.password"
+                        outlined
+                        required
+                        hint="Minimum 8 characters"
+                        persistent-hint
+                        label="Password"
+                        class="my-0 py-0"
+                        :append-icon="isPasswordShown ? 'mdi-eye' : 'mdi-eye-off'"
+                        :type="isPasswordShown ? 'text' : 'password'"
+                        :error-messages="errors[0]"
+                        @click:append="isPasswordShown = !isPasswordShown"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+                  <v-col
+                    cols="12"
+                  >
+                    <v-row justify="center">
+                      <v-btn
```

Comment:
The button design in figma has the box-shadow, is full width, and has bigger height. So you can add these properties to the btn: 
block
height="53"

and add this class: 
`v-btn--primary`

I wrote that class to override Vuetify's default primary button design; this one also added the box shadow. No need to set "color=primary" property.